crypto/stark/assets/OodsPoly.sol

```diff
@@ -65,13 +65,15 @@ contract OodsPoly \{
             }
 
             function mid_expmod(base, exponent) -> result \{
-                result := 1
-                for \{  } exponent \{ exponent := shr(exponent, 1) } \{
-                    if and(exponent, 1) \{
-                        result := mulmod(result, base, {modulus})
-                    }
-                    base := mulmod(base, base, {modulus})
-                }
+                // TOOD - Find and fix the trivial bug
+                // result := 1
+                // for \{  } exponent \{ exponent := shr(exponent, 1) } \{
+                //     if and(exponent, 1) \{
+                //         result := mulmod(result, base, {modulus})
+                //     }
+                //     base := mulmod(base, base, {modulus})
+                // }
+                result := expmod(base, exponent)
             }
```

Comment:
Let's just remove the code. Also, there should be a corresponding change in the generator to stop using it.

Reply:
i.e. here: https://github.com/0xProject/OpenZKP/blob/master/crypto/stark/src/solidity_verifier.rs#L179